### PR TITLE
Fixed styles for password protected editor map

### LINKED ISSUE #10058: Password-protection graphics look different in Windows/IE
@xavijam 

S/B 10131162

## Context
Looks like fonts & graphics aren't loading in 
Internet Explorer 8.1 / Windows 7

## Process
1. From inside a Builder map click the Privacy setting button
2. In the next screen click Privacy setting button again & choose Password
3. Enter a password, then click Publish & copy Get the url link
4. Paste link in browser tab

Happening for client in Editor map too.

## Expected result

screen_shot_2016-10-05_at_2 40 57_pm

## Current result

image

(tested in VirtualBox Windows 7 environment)

### PR DESCRIPTION
**Pull request by MariaCheca**
Fixes #10058

Added missing styles for password protected visualizations and embeds in editor.

CR @xavijam

### CODE REVIEW

File: app/views/layouts/application_password_layout.html.erb [hunk 1]
```diff
@@ -1,41 +1,30 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
   <head>
-    <meta name="viewport" content="initial-scale=1.0, user-scalable=no" />
-    <meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
-    <%= favicon_link_tag "/favicons/favicon.ico" %>
```
Comment (xavijam) [inline app/views/layouts/application_password_layout.html.erb]: Hey, the favicon! :(

**Comment by xavijam** [comment]
Things commented are important @MariaCheca 🤗

File: app/views/layouts/application_password_layout.html.erb [hunk 2]
```diff
@@ -1,41 +1,30 @@
...
 
-      <div class="foot">Contact with the owner in case you want access to it.</div>
-    </div>
+        <div class='PublicPage-header'>
+          <a href="<%= cartodb_com_hosted? ? CartoDB.url(self, 'root') : "https://carto.com" %>">
+            <img src="<%= image_path "layout/carto-logo.svg" %>" alt="CARTO">
+          </a>
+        </div>
 
-    <%= javascript_include_tag 'cdb.js' %>
```
Comment (MariaCheca) [inline app/views/layouts/application_password_layout.html.erb]: Nope, everything works fine without it :)

File: app/views/layouts/application_password_layout.html.erb [hunk 3]
```diff
@@ -1,41 +1,30 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
   <head>
-    <meta name="viewport" content="initial-scale=1.0, user-scalable=no" />
```
Comment (MariaCheca) [inline app/views/layouts/application_password_layout.html.erb]: 😱

File: app/views/layouts/application_password_layout.html.erb (hunk 1 above)
Comment (MariaCheca) [inline app/views/layouts/application_password_layout.html.erb]: 😱

**Pull request merged**